includes/class-sensei-learners-main.php

```diff
@@ -263,14 +264,50 @@ protected function get_row_data( $item ) {
 
 				if ( 'complete' == $user_activity->comment_approved || 'graded' == $user_activity->comment_approved || 'passed' == $user_activity->comment_approved ) {
 
-					$status_html = '<span class="graded">' . esc_html__( 'Completed', 'sensei-lms' ) . '</span>';
+					$progress_status_html = '<span class="graded">' . esc_html__( 'Completed', 'sensei-lms' ) . '</span>';
 
 				} else {
 
-					$status_html = '<span class="in-progress">' . esc_html__( 'In Progress', 'sensei-lms' ) . '</span>';
+					$progress_status_html = '<span class="in-progress">' . esc_html__( 'In Progress', 'sensei-lms' ) . '</span>';
 
 				}
 
+				$is_user_enrolled  = Sensei_Course::is_user_enrolled( $this->course_id, $user_activity->user_id );
+				$course_enrolment  = Sensei_Course_Enrolment::get_course_instance( $this->course_id );
+				$enrolment_results = $course_enrolment->get_enrolment_check_results( $user_activity->user_id );
+
+				$enrolment_tooltip_html = '';
+				if ( Sensei()->feature_flags->is_enabled( 'enrolment_provider_tooltip' ) ) {
```

Comment:
Could we start adding a blank line before control structures and loops? Code can feel somewhat cluttered and is harder to read, especially for those of us who need to use reading glasses. 🙁

Reply:
I added one here and below where I agree it could use some space, but I'm not sure if we want to set it as a rule to always add spaces before loops & control structures (especially when they are immediately nested or [solely] responsible for directly modifying a variable that's set right above it).